structuredevent-model/src/main/java/com/sequenceiq/environment/parameter/dto/AzureResourceEncryptionParametersDto.java

```diff
@@ -0,0 +1,37 @@
+package com.sequenceiq.environment.parameter.dto;
+
+public class AzureResourceEncryptionParametersDto {
+    private final String keyUrl;
+
+    public AzureResourceEncryptionParametersDto(AzureResourceEncryptionParametersDto.Builder builder) {
```

Comment:
This should be `private`. Only the builder shall have access to the constructor.